test/functional/lua/snippet_spec.lua

```diff
@@ -48,7 +48,7 @@ describe('vim.snippet', function()
     matches(err, pcall_err(exec_lua, string.format('vim.snippet.expand("%s")', snippet)))
   end
 
-  it('adds base indentation to inserted text', function()
+  it('adds correct base indentation to inserted text', function()
```

Comment:
Why the change here? `correct` is a bit redundant for test cases as you usually don't write a case for something to behave incorrect

Reply:
True, I just felt like I needed to update the description of this message to better reflect the added case but this change is silly.